lib/rotary-encoder/src/rotary-encoder.cpp

```diff
@@ -41,3 +45,16 @@ ISR(INT0_vect)
     }
     last_trigger_INT0 = t;
 }
+
+volatile unsigned long last_trigger_PCINT0 = 0;
+ISR(PCINT2_vect)
+{
+    unsigned long t = millis();
+    if (t - last_trigger_PCINT0)
```

Comment:
We need the exact same SW debouncing formula as for the rotation interrupt. Otherwise we can get multiple triggers on a single press.
```suggestion
    if (t - last_trigger_PCINT0 >= 2)
```

Reply:
Addressed in #27

Reply:
I agree. I don´t know why I did not add the condition.